py3status/modules/whatismyip.py

```diff
@@ -13,14 +13,8 @@
         (default 'ip')
     negative_cache_timeout: how often to check again when offline (default 2)
     timeout: how long before deciding we're offline (default 5)
-    url: change IP check url (must output a plain text IP address)
-        (default 'http://ultrabug.fr/py3status/whatismyip')
     url_geo: IP to check for geo location (must output json)
-        (default 'http://ip-api.com/json')
-
-Format placeholders:
-    {country} display the country
-    {ip} display current ip address
```

Comment:
Should I keep this list? I'm not sure. 

Also, in the new default `url_geo`, the placeholders are `{ip}` and `{country_name}` (plus many others), in the old default `url_geo` the placeholders were `{query}` and `{country}`, and in the old code it was hardcoded to replace `{ip}` with `{query}`. But we can't play smart and do such hidden replacements anymore, if we really want people to allow entering any format placeholders. 

TLDR: in the default configuration `{country}` changes to `{country_name}`, but we shouldn't do this replacement by ourselves, because if user redefines `url_geo`, on that another website the placeholder might be indeed `{country}`.